config.yml.template

```diff
@@ -5,6 +5,10 @@ email:
   # password line can be removed after OAuth is set up, if desired.
   password:
 
+amazon:
```

Comment:
please add a short comment explaining that this should be the u/p of any amazon account used for login and retrieval of trackings